### DISCUSSION TITLE
Any plans to use AnyIO instead of asyncio?

### DISCUSSION THREAD

**Discussion by markc-versa** [procrastinate-org/procrastinate#670] [Ideas]
Just wanted to ask if this has been thought about; I'm looking for projects that are compatible with trio.

**Comment by ewjoachim** [comment]
There are 2 open tickets :) https://github.com/procrastinate-org/procrastinate/issues/247 & https://github.com/procrastinate-org/procrastinate/issues/465 (I guess even I couldn't keep track of duplicates)

**Comment by markc-versa** [comment]
Woops, I thought I did a quick search earlier. Good to know :)
